src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/ResetPasswordConfirmation.cshtml

```diff
@@ -11,16 +11,9 @@
 
             <div class="nhsuk-grid-row">
                 <div class="nhsuk-grid-column-two-thirds">
-
-                    <div data-test-id="go-back-link" class="nhsuk-back-link">
-                        <a class="nhsuk-back-link__link" asp-controller="Account" asp-action="Login">
-                            <svg class="nhsuk-icon nhsuk-icon__chevron-left" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
-                                <path d="M8.5 12c0-.3.1-.5.3-.7l5-5c.4-.4 1-.4 1.4 0s.4 1 0 1.4L10.9 12l4.3 4.3c.4.4.4 1 0 1.4s-1 .4-1.4 0l-5-5c-.2-.2-.3-.4-.3-.7z"></path>
-                            </svg>
-                            Back to log in
-                        </a>
+                    <div data-test-id="go-back-section" class="nhsuk-back-link">
```

Comment:
Do we still need the `data-test-id="go-back-section"` attribute?

Reply:
👍